pages/content/amp-dev/documentation/guides-and-tutorials/integrate/embed-stories-nonamp.md

```diff
@@ -0,0 +1,553 @@
+      }
+      amp-story-player {
+        margin: 1rem auto;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <h1>
+        Page Header
+      </h1>
+    </header>
+    <h1>
+      Article Title
+    </h1>
+    <p>
+      Doggo ipsum smol wow very biscit length boy, doing me a frighten. Borking
+      doggo doggo heckin dat tungg tho, heckin good boys. Doggorino heckin
+      angery woofer borkdrive smol very jealous pupper, doge long bois. Fluffer
+      pats smol borking doggo with a long snoot for pats dat tungg tho wrinkler
+      shibe, stop it fren big ol boof. Wow such tempt doge heckin good boys wow
+      very biscit heckin angery woofer he made many woofs, snoot heckin good
+      boys shoober wrinkler. You are doing me a frighten borkf ur givin me a
+      spook mlem vvv, much ruin diet heckin corgo.
+    </p>
+    <amp-story-player style="width: 370px; height: 622px;">
+      <script type="application/json">
+        {
+          "behavior": {
+            "autoplay": true
+          }
+        }
+      </script>
+
+      <a
+        href="https://preview.amp.dev/documentation/examples/introduction/stories_in_amp/"
+      >
+      <img data-amp-story-player-poster-img src="https://amp.dev/static/samples/img/story_dog2_portrait.jpg" width="370" height="622" loading="lazy">
+        Stories in AMP - Hello World
+      </a>
+    </amp-story-player>
+    <p>
+      Such treat big ol pupper. Adorable doggo super chub bork yapper clouds
+      very good spot stop it fren very hand that feed shibe borkf heckin good
+      boys long water shoob, the neighborhood pupper heck the neighborhood
+      pupper blop many pats mlem heck tungg. noodle horse. Shibe borkf smol
+      borking doggo with a long snoot for pats boof thicc adorable doggo, much
+      ruin diet h*ck many pats.
+    </p>
+  </body>
+</html>
+```
+[/example]
+
+# Include the Web Story Player
+
+Web Stories rely on the AMP Story Player, in the form of the `<amp-story-player>` element, to embed and play stories on non-AMP sites. Implement the `amp-story-player` element by including the following two scripts in the head of your document:
+
+`<script async src="https://cdn.ampproject.org/amp-story-player-v0.js"></script>`
+
+`<link href="https://cdn.ampproject.org/amp-story-player-v0.css" rel="stylesheet" type="text/css">`
+
+The first script imports the logic for the AMP Story Player. The second script sets the default styling. 
+
+After including each script, include an `<amp-story-player>` element inside the document body.
+
+```html
+<amp-story-player style="width: 370px; height: 622px;">
+      <a
+        href="https://preview.amp.dev/documentation/examples/introduction/stories_in_amp/"
+      >
+      <img data-amp-story-player-poster-img src="https://amp.dev/static/samples/img/story_dog2_portrait.jpg" width="370" height="622" loading="lazy">
+        Stories in AMP - Hello World
+      </a>
+  </amp-story-player>
+```
+
+## Size the Web Story Player
```

Comment:
@raovs asked if we could move the attention way from the "embedding" use case of the player and more into the "full-screen" use case - keeping the immersive nature of stories while adding the extra functionality that comes with the player (like swiping to the next story).

I'll leave it up to you how we do this, but maybe we can add a line here that says: "We recommend making the player full-screen when possible, but you can also size it as you would with any other HTML element."

Reply:
I've added a few more lines, but do we have any guidance about the best way to make the WSP full-screen? I also added a bit about mobile devices - but do we also recommend full-screen on desktop experiences?

Reply:
> Do we have any guidance about the best way to make the WSP full-screen?
Hmm, we don't have documentation where we explain how to size the element to take the full screen as each publisher will have a different layout for their pages, so "fullscreen" will mean different things. 

I made a simple [codepen](https://codepen.io/maenrique/pen/vYyVjEB) that should give an idea to publishers on how to create a lightbox-like experience with the player. Maybe we can include this here?

>  do we also recommend full-screen on desktop experiences?

Yep, I would even add the `skip-to-next` button on desktop so that skipping stories is easier for users. https://github.com/ampproject/amphtml/blob/master/spec/amp-story-player.md#example-2---showing-skip-to-next-story-on-desktop